server/controllers/user.js

```diff
@@ -46,6 +46,54 @@ class UserSign {
       },
     });
   }
+
+
+  static async signin(req, res) {
+    const emailget = 'SELECT * FROM users WHERE email =$1';
+    const { rows: [emailGot] } = await pool.query(emailget, [req.body.email]);
+    if (!emailGot) {
+      return res.status(422).json({
```

Comment:
The 422 status code is not appropriate for this response. Hint: consider looking into the status code for conflicting data entries.